src/Aspire.Cli/Projects/ProjectLocator.cs

```diff
@@ -351,6 +351,12 @@ await Parallel.ForEachAsync(appHostFiles, parallelOptions, async (candidateFile,
                 logger.LogDebug("Using project file {ProjectFile}", projectFile.FullName);
                 return projectFile;
             }
+            // Handle .fsproj files
+            else if (projectFile.Extension.Equals(".fsproj", StringComparison.OrdinalIgnoreCase))
```

Comment:
Rather than duplicating the csproj branch, how about having a collection of project file extensions (csproj, fsproj, vbproj) and check whether the extension matches any of them.

Reply:
Test adjusted, support for the mentioned extensions added.